AppSettingsManagerAPI/.dev/docker-compose.yml

```diff
@@ -4,17 +4,16 @@ services:
     image: mysql:latest
     restart: always
     environment:
-      MYSQL_DATABASE: 'db'
-      MYSQL_USER: 'user'
-      MYSQL_PASSWORD: 'password'
       MYSQL_ROOT_PASSWORD: 'password'
-      MY_SQL_ROOT_HOST: '%'
+      MYSQL_ROOT_HOST: '%'
     ports:
       - '3306:3306'
     expose:
       - '3306'
     volumes:
       - my-db:/c/Temp
+      - ./sql:/docker-entrypoint-initdb.d
```

Comment:
Nice! This is pretty sweet. It's cool to not have to rerun the dotnet ef migrations script command every time and paste in the script